classes/03class/exercises/c03-docker04/mail2kvanitha/README.md

```diff
@@ -0,0 +1,121 @@
+# C03-Docker04
+
+
+## Command Execution Output
+- Commands executed to clean up both the images and the containers(running or exited) from your local:
+```
+ LIST CONTAINER DETAILS
+ > docker container ls
+   CONTAINER ID        IMAGE               COMMAND              CREATED             STATUS              PORTS                  NAMES
+   a1c08b6e0cdc        exercise02:v2       "httpd-foreground"   46 hours ago        Up 46 hours         0.0.0.0:8082->80/tcp   infallible_curran
+   2776f5b8dfc2        exercise02:v1       "httpd-foreground"   46 hours ago        Up 46 hours         0.0.0.0:8081->80/tcp   fervent_darwin
+
+ STOP THE RUNNING CONTAINERS
+ > docker container stop a1c08b6e0cdc 2776f5b8dfc2
+   a1c08b6e0cdc
+   2776f5b8dfc2
+
+ LIST IMAGE DETAILS
+ > docker images
+   REPOSITORY                    TAG                 IMAGE ID            CREATED             SIZE
+   mail2kvanitha/devopsacademy   c03-docker03-e02    e0694673648f        46 hours ago        107MB
+   exercise02                    v2                  e0694673648f        46 hours ago        107MB
+   exercise02                    v1                  bda9d1507076        47 hours ago        107MB
+   mail2kvanitha/devopsacademy   c03-docker03-e01    bda9d1507076        47 hours ago        107MB
+   da-terraform                  c03-docker-e01      c50fc0cc7915        2 days ago          56MB
+   mail2kvanitha/devopsacademy   c03-docker01        c50fc0cc7915        2 days ago          56MB
+
+ REMOVE THE IMAGE LOCALLY
+ > docker image rm exercise02:v2
+   Untagged: exercise02:v2
+
+ > docker image rm exercise02:v1
+   Untagged: exercise02:v1
+
+ > docker images
+   REPOSITORY                    TAG                 IMAGE ID            CREATED             SIZE
+   mail2kvanitha/devopsacademy   c03-docker03-e02    e0694673648f        46 hours ago        107MB
+   mail2kvanitha/devopsacademy   c03-docker03-e01    bda9d1507076        47 hours ago        107MB
+   mail2kvanitha/devopsacademy   c03-docker01        c50fc0cc7915        2 days ago          56MB
+   da-terraform                  c03-docker-e01      c50fc0cc7915        2 days ago          56MB
+
+ > docker image rm c50fc0cc7915
```

Comment:
you can use the `docker image prune` to delete all images not in use. way easier than deleting all images one by one